app/Jobs/CloneImagesOrVideos.php

```diff
@@ -253,6 +254,7 @@ private function copyImageAnnotation($volume, $copy, $selectedFileIds, $selected
         }
 
         $chunkSize = 100;
+        $databaseRecordLimit = 64999;
```

Comment:
The limit is a *parameter* limit, i.e. `records * attributes = parameters`. This means the number must be much lower here. I think you can use 10,000 here (the annotations and annotation labels have 5–6 attributes).